### PR TITLE
arm64: dts: qcom: msm8916-samsung-j5nlte: add hardware buttons

### PR DESCRIPTION
**Pull request by jugendhacker**

### CODE REVIEW

**Comment by Minecrell** [comment]
Hi @jugendhacker, CI is failing here, please take a look at the contributing guidelines: https://github.com/msm8916-mainline/linux/blob/master/CONTRIBUTING.md

File: arch/arm64/boot/dts/qcom/msm8916-samsung-j5nlte.dts [hunk 1]
```diff
@@ -268,4 +289,17 @@
...
+	gpio_keys_default: gpio_keys_default{
+		pings = "gpio107", "gpio109";
+		function = "gpio";
+
+		drive-strength = <2>;
+		bias-pull-up;
+	};
+};
+
+&pm8916_resin {
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-samsung-j5nlte.dts:302]: This should be above `&pronto`, the order is as documented in https://wiki.postmarketos.org/wiki/MSM8916_Mainlining#Device_Tree_Structure

**Comment by jugendhacker** [comment]
@Minecrell everything should be fixed now

File: arch/arm64/boot/dts/qcom/msm8916-samsung-j5nlte.dts [hunk 2]
```diff
@@ -268,4 +294,12 @@
 		drive-strength = <2>;
 		bias-disable;
 	};
+
+	gpio_keys_default: gpio_keys_default{
```
Comment (Minecrell) [inline arch/arm64/boot/dts/qcom/msm8916-samsung-j5nlte.dts:298]: Pinctrl is alphabetically ordered within `&msmgpio`, can you move it directly below accel_int?

Also:

```suggestion
	gpio_keys_default: gpio-keys-default {
```

With that it looks good to me, thanks for your effort!

**Review by Minecrell — APPROVED** [review]

**Pull request merged**